src/libs/utils.ts

```diff
@@ -72,8 +72,8 @@ const completeDateFormatParts = [
   new Intl.DateTimeFormat('default', { hour: 'numeric', minute: 'numeric' }),
 ];
 
-export const makePrettyDate = (dateString) => {
-  const date = new Date(dateString);
+export const makePrettyDate = (dateString: number | string | Date): string => {
```

Comment:
Alternatively, this type could be obtained from the Date type.

```suggestion
type DateConstructorArg = ConstructorParameters<typeof Date>;

export const makePrettyDate = (dateString: DateConstructorArg): string => {
```

Reply:
Hmm, the compiler/linter does not seem to like that one.

Reply:
Oops. `ConstructorParameters` returns an array type. So either

```ts
type DateConstructorArg = ConstructorParameters<typeof Date>[0];

export const makePrettyDate = (dateString: DateConstructorArg): string => {
```

or

```ts
type DateConstructorArgs = ConstructorParameters<typeof Date>;

export const makePrettyDate = (...args: DateConstructorArgs): string => {
  const date = new Date(...args);
```

should work.

Also, feel free to leave as you have it. The sedate formatting functions are only tangentially related to the main focus of this PR, so I don't think we need to spend a lot of time on them here.